### ISSUE TITLE
Applying for project maintaining role

### ISSUE THREAD

**Issue by answerquest** [ismyrnow/Leaflet.functionaltilelayer#9]
Hi, I've benefitted from this plugin in a project when I had started out mapping. Just revisited it with an idea in mind and saw the callout for maintainers. Would like to apply for the same. Please feel free to get in touch, ask questions etc. I'll see if I can find the exact page where I'd applied this plugin.

**Comment by answerquest** [comment]
@ismyrnow ?

**Comment by ismyrnow** [comment]
Thanks for the bump. I missed your initial message.

If you're interested in being a maintainer, I'd be willing to add you to the repository. I haven't used Leaflet in a handful of years, but if this plugin is something still useful to the Leaflet community, I'd love help with a few things:

1. Fixing examples. The tile URLs are probably outdated and need to reference a new, available tile source.
2. Looking over issues/PRs. There is a typescript definition one, and though it's probably okay to merge, there is no automation or unit tests in place to make sure stuff doesn't break. The package could probably use a minor version bump and publish to npm.
3. Upgrade to work with latest version of Leaflet. Like I said, I haven't used Leaflet in a while, so I'm not sure if this plugin still works (or is relevant) with the latest version.

I would prefer not to see the project diverge from the original intention though. If your interest is to add features or drastically expand usage, that's fine by me, but I would ask that you fork the library instead.

**Comment by answerquest** [comment]
Hi @ismyrnow , sure, will be glad to help out. I'll start off with no.3 and look into the others over the weekend.

Even I don't want the project to diverge - don't have any major changes in mind. The primary use case of being able to dynamically change the tile url as the map is browsed, is the main thing for me. It can be useful.

**Comment by ismyrnow** [comment]
Yeah, that's a great suggestion, which would be backwards compatible. I could see that being very useful.

**Comment by iilei** [comment]
I would also like to help maintaining this project.

Thought of publishing it via npm but not wanting to steal it from you 😄
